AudibleApiDTOs/ContentLicenseDtoV10.cs

```diff
@@ -125,35 +139,26 @@ private static VoucherDtoV10 DecryptLicenseResponse(ContentLicenseDtoV10 content
             byte[] key = new byte[16];
             byte[] iv = new byte[16];
 
-            using (var sha256 = System.Security.Cryptography.SHA256.Create())
-            {
-                sha256.ComputeHash(keyComponents);
-                Array.Copy(sha256.Hash, 0, key, 0, 16);
-                Array.Copy(sha256.Hash, 16, iv, 0, 16);
-            }
+            using var sha256 = System.Security.Cryptography.SHA256.Create();
+            sha256.ComputeHash(keyComponents);
+            Array.Copy(sha256.Hash, 0, key, 0, 16);
+            Array.Copy(sha256.Hash, 16, iv, 0, 16);
 
             var cipherText = Convert.FromBase64String(contentLicense.ContentLicense.LicenseResponse);
 
             string plainText;
 
-            using (var aes = System.Security.Cryptography.Aes.Create())
-            {
-                aes.Mode = System.Security.Cryptography.CipherMode.CBC;
-                aes.Padding = System.Security.Cryptography.PaddingMode.None;
+            using var aes = System.Security.Cryptography.Aes.Create();
+            aes.Mode = System.Security.Cryptography.CipherMode.CBC;
+            aes.Padding = System.Security.Cryptography.PaddingMode.None;
+
+            using var decryptor = aes.CreateDecryptor(key, iv);
 
-                using (var decryptor = aes.CreateDecryptor(key, iv))
+            using var csDecrypt = new System.Security.Cryptography.CryptoStream(new System.IO.MemoryStream(cipherText), decryptor, System.Security.Cryptography.CryptoStreamMode.Read);
 
-                using (var msDecrypt = new System.IO.MemoryStream(cipherText))
```

Comment:
Is "using" no longer needed on this stream?

Reply:
No, I'm almost certain it gets disposed with csDecrypt. I used the same code in AAXClean where it gets called over 100,000 timer per second and there's no memory leak.

Reply:
Verified it is disposed with csDecrypt.